src/components/chats/chat/ChatMessages.vue

```diff
@@ -152,19 +184,34 @@ export default {
     prevRoomUuid: null,
   }),
 
+  mounted() {
+    window.addEventListener('online', () => {
+      this.resendMessages();
+    });
+
+    // const observer = new IntersectionObserver((entries) => {
```

Comment:
Dead code?

Reply:
No, this code is commented to handle future updates with the backend